src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -191,11 +193,11 @@ function prepareMCQQuestions() {
     var radioButtons = {};
     var radioStates = {};
 
-    for (var i = 0; i < mcqQuestionNums.length; i++) {
+    $.each(mcqQuestionNums, function(i) {
         var qnNum = mcqQuestionNums[i];
         var numResponses = $('[name="questionresponsetotal-' + qnNum + '"]').val();
 
-        for (var j = 0; j < numResponses; j++) {
+        $.each(numResponses, function(j) {
```

Comment:
So I changed this to `$.each(new Array(numResponses), ...` and discovered that it doesn't work. Changing back to the for loop version made everything work perfectly.

Upon further digging, I found out that
```js
$.each(new Array(numResponses), ...)
(new Array(numResponses)).forEach(...
```
both screw up differently. This is weird...